inventory/sample/group_vars/all.yml

```diff
@@ -56,6 +56,15 @@ extra_agent_args: >-
 # image tag for kube-vip
 kube_vip_tag_version: "v0.6.4"
 
+# loadbalancer to use for services: "kube-vip" or "metallb"
+service_lb_type: "metallb"
+
+# tag for kube-vip-cloud-provider manifest
+kube_vip_cloud_provider_tag_version: "main"
+
+# kube-vip ip range for load balancer
+kube_vip_lb_ip_range: "192.168.30.80-192.168.30.90"
```

Comment:
I was thinking instead of having `service_lb_type` we could just check to see if `kube_vip_lb_ip_range` had values or `metal_lb_ip_range` had values and infer which one to use based on that?
e.g. in this scenario `kube-vip` would be used.

```yaml
# metallb ip range for load balancer
#metal_lb_ip_range: "192.168.30.80-192.168.30.90"

# kube-vip ip range for load balancer
kube_vip_lb_ip_range: "192.168.30.80-192.168.30.90"
```

Then the default group vars could be:

```yaml

# metallb ip range for load balancer
metal_lb_ip_range: "192.168.30.80-192.168.30.90"

# kube-vip ip range for load balancer
#kube_vip_lb_ip_range: "192.168.30.80-192.168.30.90"
```

This way someone would only need to uncomment `kube-vip` and comment `metal lb` and everything would work, we don't break the contract, and we don't need a type.  Does that make sense?

Reply:
I simplified the configuration according to your suggestion